cli/wildcat-codemod.js

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env babel-node
+
+import {echo, exit, pwd} from "shelljs";
+import nomnom from "nomnom";
```

Comment:
it's the args parser
https://github.com/harthur/nomnom

same one that jscodeshift uses 
https://github.com/facebook/jscodeshift#options